package.json

```diff
@@ -279,6 +279,11 @@
         "title": "Run all tests",
         "when": "resourceExtname == .wtest",
         "category": "Wollok"
+      },
+      {
+        "command": "wollok.init.project",
+        "title": "Initialize new Wollok project in the workspace directory",
+        "category": "Wollok"
```

Comment:
Esta opción solo aparece si estás parado sobre un archivo Wollok? 
O si levantas el VSCode en un directorio vacío ya se puede tirar?

Reply:
@PalumboN , ahí lo probé en una carpeta vacía

![wollokInit](https://github.com/user-attachments/assets/7c02d59f-d5cf-48fb-a3a6-a4597f32ab36)

Después le di Run in REPL y anduvo joya

![image](https://github.com/user-attachments/assets/e44e90aa-6dd0-4e5c-9d7a-a85faa851646)

Pero por ahí estaría bueno que una vez que hagas init se restartee el VSC dentro de la carpeta creada. Restartear puede hacer con este comando

```ts
vscode.commands.executeCommand("workbench.action.reloadWindow")
```

pero la que va es

```ts
vscode.commands.executeCommand("workbench.action.files.openFolderInNewWindow")
```

Reply:
eso te abre una ventana de diálogo para que selecciones la carpeta